appscale/tools/remote_helper.py

```diff
@@ -259,6 +284,25 @@ def spawn_nodes_in_cloud(cls, options, count=1):
 
 
   @classmethod
+  def spawn_other_nodes_in_cloud(cls, options, agent, params, count=1):
+    """Starts a single virtual machine in a cloud infrastructure.
+
+    This method also prepares the virual machine for use by the AppScale Tools.
```

Comment:
virual -> virtual and the same on top for the spawn_load_balancers_in_cloud method (Line 266)